etc/update_version.sh

```diff
@@ -28,7 +28,7 @@ echo "Setting version to $relvers, released $reldate_iso"
 
 # update version in several files
 perl -pi -e 's;version = "[^"]+";version = "'$relvers'";' Project.toml
-perl -pi -e "s;Version [^ }]+;Version $relvers;" README.md
+perl -pi -e "s;Version [^ },]+;Version $relvers;" README.md
```

Comment:
```suggestion
perl -pi -e "s;Version [^ },]+;Version $relvers,;" README.md
```
? Not sure. Let @fingolfin review this here.

Reply:
It will capture (+replace) only the version, not including the subsequent comma (or `}` or ` ` for different places in the readme), so no need to add another comma after the new version number.